duckduckgo/macos-browser
# remove www dropping from suggestions (macOS)

Task/Issue URL: https://app.asana.com/0/230839424936/1204396770411787/f

**Description**: Updates BSK version where (www dropping from suggestions is removed)

**Steps to test this PR**:
1. on the search bar type softether you should get a softether.org suggestion (with the globe) click on it and check it loads the website.



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
